packages/protobuf/src/reflect/registry.ts

```diff
@@ -829,7 +829,7 @@ function newField(
           keyField.scalar != ScalarType.FLOAT &&
           keyField.scalar != ScalarType.DOUBLE,
       );
-      const valueField = mapEntry.fields.find((f) => f.proto.number === 2);
+      const valueField = mapEntry.fields.find((f) => f.number === 2);
```

Comment:
We can use `DescField.number` here, no need to go through `FieldDescriptorProto`.